examples/platforms/nrf528xx/src/radio.c

```diff
@@ -444,13 +453,17 @@ otError otPlatRadioReceive(otInstance *aInstance, uint8_t aChannel)
 
     bool result;
 
+    sCurrentChannel = aChannel;
```

Comment:
Can we use the function `nrf_802154_channel_get` to replace the variable `sCurrentChannel `?